src/appShell/App/Container.tsx

```diff
@@ -65,6 +62,27 @@ export default class Container extends React.Component<IContainerProps, {}> {
             );
         }
 
+        if (ServerConfigHelpers.oncoKbIsEnabledWithoutToken()) {
+            return (
+                <div className="contentWrapper">
+                    <ErrorScreen
+                        title={'OncoKB token is not available'}
+                        body={
+                            <p>
+                                As of version 3.2.1, all cBioPortal
+                                installations require an access token to use the latest OncoKB data.
+                                You should either disable OncoKB or register an OncoKB account.
+                                Please review these instructions for how to do so.{' '}
+                                <a href="https://github.com/cBioPortal/cbioportal/blob/master/docs/OncoKB-Data-Access.md">
+                                    https://github.com/cBioPortal/cbioportal/blob/master/docs/OncoKB-Data-Access.md
```

Comment:
- Do not merge this PR, before adding the actual documentation.
- Is this the right location for this documentation?

Reply:
might be nice to directly mention the properties here as well i.e. `show.oncokb` and `oncokb.token`.

The documentation should prolly point to some link at docs.cbioportal.org instead of the markdown directly. That website is prettier :). Could put the anchor on the `instructions` instead of showing full url

We should change the version prolly. 3.2.1 is already released. We can change it right before merging just made  a comment so we remember to do so

Reply:
keeping the full url for consistency: https://github.com/cBioPortal/cbioportal-frontend/blob/05017f04f977acd6ef916fe54362ff1627ef35fe/src/appShell/App/Container.tsx#L56